.github/workflows/cli-release.yml

```diff
@@ -53,6 +53,8 @@ jobs:
         run: make cli.prepare
         env:
           RILL_UI_PUBLIC_PYLON_APP_ID: "26a0fdd2-3bd3-41e2-82bc-1b35a444729f"
+          RILL_UI_PUBLIC_INTAKE_USER: "data-modeler"
+          RILL_UI_PUBLIC_INTAKE_PASSWORD: ${{ secrets.RILL_INTAKE_PASSWORD }}
```

Comment:
did we add this password already?

Reply:
No. Pinged you for location, lets take it on slack.